### DISCUSSION TITLE
Has anyone had issues with some photos not getting uploaded to immich?

### DISCUSSION THREAD

**Discussion by untenops** [immich-app/immich#775] [Q&A]
I've been trying immich for a little over month now an have been very happy with it.  Really liking it but was wondering if maybe I'm doing something wrong or maybe there is a bug.  Every once in a while some photos I take get completely ignored by the app.  They don't show up in the web interface, and the app never shows them.  But if I go to the Backup settings in the app and double check under Backup Albums, and look in the Camera folder there, the app see the pictures.  But the main page still does not recognize them.

**Comment by alextran1502** [comment]
Do you know which type of photo is it? We probably will need more context of the issue

**Comment by untenops** [comment]
Sure I'll try to answer any questions.  The last few picture I took with my phone are not showing up in immich.  They are showing up in Google photos, and on my nextcloud, all jpg.  Today I open the immich app to check and the pictures I took recently are not shown in the app.  Had a few photo also get lost a few weeks ago.  Never showed up in the app.    The App and server are both on the latest versions. 

![Screenshot_20221003-134056](https://user-images.githubusercontent.com/4633453/193669604-bac40440-b9e8-40ee-b5c3-9eb1ff80c173.png)

Here's a shot of the apps main page scrolled all the way to the top, last photo shown on Sep 27th

![Screenshot_20221003-134005](https://user-images.githubusercontent.com/4633453/193669756-80951ff8-8323-4cbc-97ad-e36176b17cef.png)

And then in the app on the Select Album page, if I open the folder, I can see the missing photos. 

![Screenshot_20221003-134036](https://user-images.githubusercontent.com/4633453/193670031-c0f6c4fa-5852-4ea4-89ed-95f5ce1b05c6.png)

**Reply by alextran1502** [reply]
What is the status of the backup page? Does it say any remaining photos in there?

**Reply by untenops** [reply]
No. under Remainder it shows 0. 

![Screenshot_20221003-151433](https://user-images.githubusercontent.com/4633453/193683998-576786f6-6b47-400b-9775-ae82aef52738.png)

**Reply by alextran1502** [reply]
Which docker tag do you use for the container images? `release` or `latest`?

**Reply by alextran1502** [reply]
can you please open an issue and provide the required information so we can track this bug?

**Reply by untenops** [reply]
Using release.
